dist/images/env-check.sh

```diff
@@ -0,0 +1,85 @@
+#!/bin/bash
+
+set +e
+
+echo "Start environment check"
+echo "***************************"
+echo "1) check cni configuration"
+if [ ! -e "/etc/cni/net.d" ]; then
+  echo "Directory /etc/cni/net.d does not exist, please check kube-ovn-cni pod status"
+fi
+for file in $(ls "/etc/cni/net.d")
+do
+  if [[ ! $file =~ "kube-ovn.conflist" ]]; then
+    echo "Check files in /etc/cni/net.d, make sure if the config file $file should be deleted"
+  fi
+done
+
+echo "2) check system ipv4 config"
+probe_mtu=`cat /proc/sys/net/ipv4/tcp_mtu_probing`
+if [ $probe_mtu == 0 ]; then
+  echo "The 'tcp_mtu_probing' config may affect traffic, make sure if /proc/sys/net/ipv4/tcp_mtu_probing should be set to 1"
+fi
+if [ -e /proc/sys/net/ipv4/tcp_tw_recycle ]; then
+  recycle=`cat /proc/sys/net/ipv4/tcp_tw_recycle`
+  if [ $recycle == 1 ]; then
+    echo "The 'tcp_tw_recycle' config affects nodeport service, make sure change /proc/sys/net/ipv4/tcp_tw_recycle to 0"
+  fi
+fi
+
+echo "3) check checksum value"
+which netstat
```

Comment:
Use `which netstat 2>/dev/null >/dev/null` to suppress the output.

Reply:
modified